VAMobile/src/translations/en/common.json

```diff
@@ -1386,4 +1386,4 @@
   "whatsNew.bodyCopy.2.13": "You can now show proof of your Veteran status with the app.",
   "whatsNew.dismissMessage": "Dismiss this message",
   "whatsNew.title": "What's new"
-}
+}
```

Comment:
need to add a new line it looks like

Reply:
I did not actively delete the new line. I noticed it was missing, added it back in, hit save, and it is automatically deleted by prettier.